### PR TITLE
produce events for readyz going true

### PR DESCRIPTION
**Pull request by deads2k**
trying to debug when our kube-apiservers are going readyz=true

### CODE REVIEW

**Comment by deads2k** [comment]
readyz events are  working, associated by master

/retest

/test e2e-aws-upgrade

**Comment by deads2k** [comment]
/hold

integration is borked somehow

File: staging/src/k8s.io/apiserver/pkg/server/config.go [hunk 1]
```diff
@@ -588,7 +588,7 @@ func eventReference() (*corev1.ObjectReference, error) {
 	}
 	if len(ns) == 0 {
 		pod = ""
-		ns = "kube-system"
+		ns = "openshift-kube-apiserver"
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/server/config.go:591]: @s-urbaniak @sttts using kube-system through off an event integration test.  Also, we know that our kube-apiserver is never in kube-system.  Oddly, this entire method appears to be carried (stefan I think?), but it's not in a patch file.

Comment (sttts) [reply staging/src/k8s.io/apiserver/pkg/server/config.go:591]: :+1: for changing this.

@mfojtik is this your code?

Comment (mfojtik) [reply staging/src/k8s.io/apiserver/pkg/server/config.go:591]: @sttts negative

**Comment by deads2k** [comment]
> /hold
> 
> integration is borked somehow

/hold cancel

got it worked out.

**Comment by sttts** [comment]
/lgtm
/approve

Tests are red.

**Comment by deads2k** [comment]
failures appear unrelated atm

**Pull request merged**